src/typecheck.c

```diff
@@ -914,9 +949,9 @@ static ExpressionTypes *typecheck_expression(FileTypes *ft, const AstExpression
         if (temptype->kind != TYPE_CLASS)
             fail_with_error(
                 expr->location,
-                "left side of the '.' operator must be a class, not %s",
+                "left side of the '.' operator must be a class instance, not %s",
```

Comment:
"class instance" just sounds weird. Could probably grep for it, and replace it with "instance of a class"